quotalib/qutoalib.go

```diff
@@ -9,12 +9,38 @@ import (
 	"io/ioutil"
 	"bytes"
 	"encoding/json"
+	"../types"
 )
 
+type QInfos struct{
+	Infos []subInfo "json:'info'"
+	
+}
+
+type subInfo struct{
+	Guarantee []subsubInfo "json:'guarantee'"
+	Role string "json:'role'"
+	
+}
+
+
+type subsubInfo struct{
```

Comment:
Why `subsubinfo` as against `SubsubInfo`?  Plus Subsubifo does not look right to me.  Lets call this QuotaInfo